### PR TITLE
craft-oa/gdpr-invitation#71 add accept invitation cancel popup text changes

### PR DESCRIPTION
**Pull request by ipula**
see https://github.com/craft-oa/gdpr-invitation/issues/71 and https://github.com/craft-oa/gdpr-invitation/issues/68

### CODE REVIEW

File: locale/en/invitation.po [hunk 1]
```diff
@@ -356,4 +356,16 @@ msgid "acceptInvitation.privacyStatement.validation"
 msgstr "Please confirm that you have read and agree privacy statement"
 
 msgid "acceptInvitation.cancel.message"
-msgstr "Are you sure want to cancel accepting invitation?"
+msgstr "Are you sure you want to cancel? Canceling now will stop the role acceptance process, and you’ll need to restart from the invitation email to accept the role again.If you’re already a user, you’ll be taken back to the dashboard. If not, you’ll need to access the invitation email to start the process again."
```
Comment (Tribunal33) [inline locale/en/invitation.po:359]: This "...again.If you’re already..." should be "...again. If you’re already..." <-- there is an extra space after . and before "if".  

Is there a mock up for this?

**Review by Tribunal33 — CHANGES_REQUESTED** [review]

File: locale/en/invitation.po (hunk 1 above)
Comment (ipula) [reply locale/en/invitation.po:359]: yes , Please check @Devika008 comment on this https://github.com/craft-oa/gdpr-invitation/issues/68

Comment (Tribunal33) [reply locale/en/invitation.po:359]: @ipula it looks good except for the missing space that I could see from the mock up.  I got @Devika008 to add the missing space to the [gdpr-invitation#68](https://github.com/craft-oa/gdpr-invitation/issues/68) issue as well.

**Comment by withanage** [comment]
@Tribunal33  when you are ok with this, I can merge this one.

**Comment by Tribunal33** [comment]
Looks good @withanage merge away.

**Pull request merged**